### DISCUSSION TITLE
2021-11-01 Core Team Update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#117] [Core]
# Highlights:
Nearcon & team offsite last week

# Done:
@Longarithm : 
* PR that will remove the runtime config from genesis config (https://github.com/near/nearcore/pull/5064)
* analysis of the mainnet contract compilation costs (being discussed in Zulip + https://github.com/near/nearcore/pull/4733#issuecomment-954919299)

@mzhangmzz : Fixes related to sharding deployment: 
* memory issue (https://github.com/near/nearcore/pull/5032)
* garbage collection issue (https://github.com/near/nearcore/pull/5040)
* bug in shard_tries: (https://github.com/near/nearcore/pull/5058)
* panic if node tries to sync just after sharding upgrade (https://github.com/near/nearcore/pull/5067)

@mzhangmzz : created document with post-mortem for testnet launch and pre-mortem for mainnet launch 

@pmnoxx : PR for moving the routing computation to a separate thread (https://github.com/near/nearcore/pull/5089)

# Progress:

@mzhangmzz : fix few small bugs after testnet launch & discuss the plan for the mainnet launch
@pmnoxx : working on the network rate limiter and start measuring memory usage on each peer.

@mm-near : documenting how our networking stack works

@abacabadabacaba - wrote first version of more efficient state representation (https://github.com/near/nearcore/discussions/4815)

**Comment by maxzaver** [comment]
> @Longarithm : PR that will remove the runtime config from genesis config
> @Longarithm: analysis of the mainnet contract compilation costs (current fees don't reflect real operations well)

@mm-near could we please link the PRs? I wasn't able to find them.

**Reply by Longarithm** [reply]
@nearmax I added the links.
